### PR TITLE
Locator added in host view

### PR DESCRIPTION
**Pull request by amolpati30**
This locator helps to inherit the compute profile option.

Dependent PR: https://github.com/SatelliteQE/robottelo/pull/15488

### CODE REVIEW

**Review by LadislavVasina1 — APPROVED** [review]
Locators look good to me

File: airgun/views/computeresource.py [hunk 1]
```diff
@@ -440,6 +443,7 @@ class storage(RemovableWidgetsItemsListView):
             add_item_button = Text(
                 "//button[contains(concat(' ', @class, ' '), ' btn-add-controller ')]"
             )
+            data_store = FilteredDropdown(id="s2id_autogen41")
```
Comment (shweta83) [inline airgun/views/computeresource.py:446]: Does this have any other locator? This looks like a temporary one.

Comment (amolpati30) [reply airgun/views/computeresource.py]: change the temporary locator.

**Comment by amolpati30** [comment]
"trigger": "test-robottelo"
"pytest": "-k 'test_positive_provision_end_to_end' tests/foreman/ui/test_computeresource_vmware.py"
"airgun": "15488"

**Comment by amolpati30** [comment]
"trigger": "test-robottelo"
"pytest": "-k 'test_positive_provision_end_to_end' tests/foreman/ui/test_computeresource_vmware.py"
"robottelo": "15488"

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 325
Build Status: SUCCESS
PRT Comment: pytest /opt/app-root/src/robottelo/tests/foreman -v -m build_sanity --external-logging
Test Result : ======= 12 passed, 5527 deselected, 5636 warnings in 1877.34s (0:31:17) ========
```

**Review by Gauravtalreja1 — APPROVED** [review]

**Review by shweta83 — APPROVED** [review]
Ack

**Pull request merged**